src/en/news/blog/2022/mclock-vs-wpq-testing-with-background-ops-part2/index.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "Comparison of WPQ and mClock Schedulers with Background and Recovery operations with Emphasis on Scrub Operations"
```

Comment:
Sridhar's title is "QoS in Quincy: Comparison of mClock and WPQ schedulers with background operations - Part 1", so we can modify this one to match it:
```suggestion
title: "QoS in Quincy: Comparison of mClock and WPQ schedulers with background and recovery operations - Part 2"
```